phylib/io/model.py

```diff
@@ -327,7 +327,8 @@ def __init__(self, **kwargs):
         elif not isinstance(self.dat_path, (list, tuple)):
             self.dat_path = [self.dat_path]
         assert isinstance(self.dat_path, (list, tuple))
-        self.dat_path = [Path(_).resolve() for _ in self.dat_path]
+        print(self.dat_path)
```

Comment:
is this print statement a debug leftover or a real indication to the user?

Reply:
Definitely a leftover debug! Sorry I missed that